package.json

```diff
@@ -11,6 +11,7 @@
   },
   "scripts": {
     "test": "mocha test",
+    "lint": "tslint -c tslint.json src/**/*.ts",
```

Comment:
The `blockchain` and `server` folders are both deeper than this.  Does `tslint` have a recursive flag?